internal/uvm/create_lcow.go

```diff
@@ -84,8 +84,8 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		SCSIControllerCount:   1,
 		UseGuestConnection:    true,
 		ExecCommandLine:       fmt.Sprintf("/bin/gcs -log-format json -loglevel %s", logrus.StandardLogger().Level.String()),
-		ForwardStdout:         true,
-		ForwardStderr:         false,
+		ForwardStdout:         false,
+		ForwardStderr:         true,
```

Comment:
Was this a regression? Do we need a gcs change for the logrus output here?

Reply:
It regressed as part of refactoring work. GCS outputs logs to stderr, so that should be forwarded by default.

https://github.com/Microsoft/hcsshim/commit/d144c3909d961ceda03c8a7fb749968ba4d0d96f#diff-d9bb733a142be5ae9865356ef8f98459

Reply:
Ok sweet. LGTM